packages/sdk/src/config/localConfig.ts

```diff
@@ -141,6 +141,7 @@ export const localConfig: RawConfig = {
       nativeTokenDecimals: BigInt(18),
       blockConfirmations: 1,
       bridge: '0x',
+      parachainId: 0, // ? what is the local parachain id
```

Comment:
I don't think this is local parachain but rather local version of config. So this should be real parachain id

Reply:
```suggestion
      parachainId: 0, // local setup is not parachain
```